src/main/antlr/LexBasic.g4

```diff
@@ -131,7 +130,7 @@ fragment CharLiteral	: SQuote ( EscSeq | ~['\r\n\\] )  SQuote	;
 fragment SQuoteLiteral	: SQuote ( EscSeq | ~['\r\n\\] )* SQuote	;
 fragment DQuoteLiteral	: DQuote ( EscSeq | ~["\r\n\\] )* DQuote	;
 fragment USQuoteLiteral	: SQuote ( EscSeq | ~['\r\n\\] )* 			;
-
+DOT:'.';
```

Comment:
Why did you need to add this exactly?

Reply:
Oh I see, it was to remove a build error. I don't think it should be added here, because it's already defined in `STGLexer.g4`. The correct solution would be to configure the ANTLR4 Gradle plugin to only code for non-included grammars, like this:

```groovy
sourceSets {
    main.java.srcDirs += ['src/gen/java']
    main.antlr.includes = ["ST*.g4"]
}
```